Goobi/config/log4j.properties

```diff
@@ -31,7 +31,7 @@ log4j.appender.stdout.layout.ConversionPattern=%-5p %d %C.%M(%F:%L)%n        %m%
 
 # ein Rolling-File-Appender (10 Backups)
 log4j.appender.rolling=org.apache.log4j.RollingFileAppender
-log4j.appender.rolling.File=/usr/local/logs/goobi.log
+log4j.appender.rolling.File=/usr/local/kitodo/logs/goobi.log
```

Comment:
Name of log file should kitodo.log. I missed that. `build.xml` contains log file entries? I think this entries could be removed as no log file should be created inside source tree.